src/utils.rs

```diff
@@ -0,0 +1,22 @@
+use std::{env, fs::File, io};
+
+const VERIFICATION_KEY_FILE_GITHUB: &str = "https://raw.githubusercontent.com/matter-labs/era-contracts/main/tools/data/scheduler_key.json";
+
+pub async fn check_should_download_verification_key(update_verification_key: Option<bool>) {
```

Comment:
nit: rename to "update_verification_key_if_needed".

(the "check" prefix suggests that it is a function that is stateless and just returns the boolean)